src/utils/uuidValidator.js

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
You can simply use express-validator `isUUID(4)` here. There won't be any need for this helper

Reply:
Noted